File: app/src/main/java/com/multimedia/writeyourthink/SplashActivity.java
package com.multimedia.writeyourthink;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.util.Log;

public class SplashActivity  extends Activity {

    int[] activites = new int[] {1,2};
    int actRand = 0;

    @Override
    protected void onCreate(Bundle savedInstanceStare) {
        super.onCreate(savedInstanceStare);




        actRand = (int) (Math.random() * activites.length)+1;
        Log.d("Lee", "result :" + actRand);
        switch (actRand) {
            case 1 :
                Intent intent2 = new Intent(SplashActivity.this, SplashActivity2.class);
                startActivity(intent2);
                finish();
                break;
            case 2 :
                Intent intent = new Intent(SplashActivity.this, SplashActivity3.class);
                startActivity(intent);
                finish();
                break;

        }
    }
}

File: app/src/main/java/com/multimedia/writeyourthink/SplashActivity.kt
package com.multimedia.writeyourthink

import android.app.Activity
import android.os.Bundle
import android.content.Intent
import android.util.Log
import com.multimedia.writeyourthink.SplashActivity2
import com.multimedia.writeyourthink.SplashActivity3

class SplashActivity : Activity() {
    var activites = intArrayOf(1, 2)
    var actRand = 0
    override fun onCreate(savedInstanceStare: Bundle?) {
        super.onCreate(savedInstanceStare)
        actRand = (Math.random() * activites.size).toInt() + 1
        Log.d("Lee", "result :$actRand")
        when (actRand) {
            1 -> {
                val intent2 = Intent(this@SplashActivity, SplashActivity2::class.java)
                startActivity(intent2)
                finish()
            }
            2 -> {
                val intent = Intent(this@SplashActivity, SplashActivity3::class.java)
                startActivity(intent)
                finish()
            }
        }
    }
}